samples/dotnet-svcutil.xmlserializer-instructions.md

```diff
@@ -0,0 +1,80 @@
+ # Using svctuil.xmlserializer on .NET Core
+
+Just like the svcutil XmlSerializer Type Generation function on desktop, dotnet-svcutil.xmlserializer NuGet package is the solution for .NET Core and .NET Standard Libraries. It pre-generates c# serialization code for the types used by Service Contract in the client applications that can be serialized using the XmlSerializer to improve the startup performance of Xml Serialization when serializing or de-serializing objects of those types using XmlSerializer. 
```

Comment:
Please add a link to the NuGet package.

Reply:
The package still not release yet so the link doesn't exist now. Will add it later.